src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -32,11 +32,13 @@ class ExceptionListener implements EventSubscriberInterface
 {
     protected $controller;
     protected $logger;
+    protected $debug;
```

Comment:
should have been private (too bad that I see it just *after* the release is done)